src/lib/polygon.js

```diff
@@ -31,7 +31,11 @@ var polygon = module.exports = {};
  *          returns boolean: is pt inside the polygon (including on its edges)
  */
 polygon.tester = function tester(ptsIn) {
-    if(Array.isArray(ptsIn[0][0])) return polygon.multitester(ptsIn);
+
+    // Throw an error if any code tries to pass this no
+    // longer supported argument shape. The local multitester became
+    // obsolete when introducing click-to-select in PR #2944.
+    if(Array.isArray(ptsIn[0][0])) throw new Error('multitester called!');
```

Comment:
I guess this is the safest way to go if we're not confident enough to remove this entirely just yet - or perhaps even safer would be a `Lib.warn` so the whole plot doesn't freeze, just selection or hover breaks? @etpinard thoughts? Seems to me if we do leave this here, we should make an issue and tag it with `v1.42` or `v1.43` to remove this block.

Reply:
I'm voting for :hocho: this thing entirely. I'm 99.99% percent confident this won't break anything.

Reply:
:hocho: Removed it entirely.